.github/labeler.yml

```
@@ -28,27 +28,42 @@
   - "**/CMakeLists.txt"
   - "**/*.cmake"
 
+"Crafting / Construction / Recipes":
+  - "**/recipes/**/*"
+  - data/json/construction.json
+  - data/json/construction_category.json
+  - data/json/construction_group.json
+  - data/json/deconstruction.json
+
 "NPC / Factions":
-  - data/json/npcs/**/*
+  - "**/npcs/**/*"
+
+"Appliance/Power Grid":
+  - "**/appliance/**/*"
```

```suggestion
  - "**/appliance**"
```
Makes sense to use this in this case 'cause there is an appliance_s_.json file and folder somewhere